AstSemantics.md

```diff
@@ -530,3 +530,5 @@ In the MVP, `has_feature` would always return false. As features were added post
 `has_feature` would start returning true. `has_feature` is a pure function, always
 returning the same value for the same string over the lifetime of a single
 instance and other related (as defined by the host environment) instances.
```

Comment:
"function" in the mathematical sense, not literally a wasm function.  I don't know what you mean about s-expression children, but `has_feature` is nullary, so it wouldn't have children.